azext_edge/edge/params.py

```diff
@@ -302,6 +310,822 @@ def load_iotops_arguments(self, _):
+        )
+        context.argument(
+            "scope",
+            options_list=["--scope"],
+            help="Resource identifier (application ID URI) of the resource, affixed with the .default suffix.",
+            arg_group="User Assigned Managed Identity",
+        )
+        context.argument(
+            "tenant_id",
+            options_list=["--tenant-id", "--tid"],
+            help="The tenant ID of the user assigned identity.",
+            arg_group="User Assigned Managed Identity",
+        )
+        context.argument(
+            "no_auth",
+            options_list=["--no-auth"],
+            arg_type=get_three_state_flag(),
+            help="No authentication for the endpoint.",
+        )
+        context.argument(
+            "latency",
+            options_list=["--latency", "-l"],
+            help="The batching latency in seconds. Min value: 0, max value: 65535.",
+            type=int,
+            arg_group="Batching Configuration",
+        )
+        context.argument(
+            "message_count",
+            options_list=["--message-count", "--mc"],
+            help="Maximum number of messages in a batch. Min value: 0, max value: 4294967295.",
+            type=int,
+            arg_group="Batching Configuration",
+        )
+        context.argument(
+            "audience",
+            options_list=["--audience", "--aud"],
+            help="Audience of the service to authenticate against.",
+            arg_group="System Assigned Managed Identity",
+        )
+        context.argument(
+            "acks",
+            options_list=["--acks"],
+            arg_type=get_enum_type(
+                DataflowEndpointKafkaAcksType,
+                default=DataflowEndpointKafkaAcksType.ALL.value
+            ),
+            help="Level of acknowledgment from the Kafka broker to "
+            "ensure that the message sent by producer is successfully "
+            "written to the topic and replicated across the Kafka cluster.",
+        )
+        context.argument(
+            "batching_disabled",
+            options_list=["--batching-disabled", "--bd"],
+            arg_type=get_three_state_flag(),
+            help="Disable batching.",
+            arg_group="Batching Configuration",
+        )
+        context.argument(
+            "max_byte",
+            options_list=["--max-byte", "--mb"],
+            help="Maximum number of bytes in a batch.",
+            type=int,
+            arg_group="Batching Configuration",
+        )
+        context.argument(
+            "cloud_event_attribute",
+            options_list=["--cloud-event-attribute", "--cea"],
+            arg_type=get_enum_type(
+                KafkaCloudEventAttributeType,
+                default=KafkaCloudEventAttributeType.PROPAGATE.value,
+            ),
+            help="CloudEvent settings type to map events to cloud. "
+            "Different message format are required by different setting.",
```

Comment:
Different message format**s** are required by different setting**s**

is there a user accessible mapping for message format -> setting?

Reply:
yeah there is, adding aka.ms link in help document